### PR TITLE
Update version.go

### PR DESCRIPTION
**Pull request by marcin-ozog**
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/48302

### CODE REVIEW

**Review by SzymonKowalczyk — CHANGES_REQUESTED** [review]
Please review old commit: https://github.com/IBM/ibm-licensing-operator/commit/245526930c1a924bf32bd1137034d84da03004b2

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -365,15 +365,15 @@ spec:
                       - ibm-licensing-operator
                     env:
                       - name: IBM_LICENSING_IMAGE
-                        value: quay.io/opencloudio/ibm-licensing:1.6.0
+                        value: quay.io/opencloudio/ibm-licensing:1.7.0
                       - name: IBM_LICENSE_SERVICE_REPORTER_UI_IMAGE
-                        value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.6.0
+                        value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.7.0
                       - name: IBM_POSTGRESQL_IMAGE
                         value: quay.io/opencloudio/ibm-postgresql:12.0.7
```
Comment (SzymonKowalczyk) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:372]: set value to 12.0.8

File: config/manager/manager.yaml [hunk 2]
```diff
@@ -35,15 +35,15 @@ spec:
             - ibm-licensing-operator
           env:
             - name: IBM_LICENSING_IMAGE
-              value: quay.io/opencloudio/ibm-licensing:1.6.0
+              value: quay.io/opencloudio/ibm-licensing:1.7.0
             - name: IBM_LICENSE_SERVICE_REPORTER_UI_IMAGE
-              value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.6.0
+              value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.7.0
             - name: IBM_POSTGRESQL_IMAGE
               value: quay.io/opencloudio/ibm-postgresql:12.0.7
```
Comment (SzymonKowalczyk) [inline config/manager/manager.yaml:42]: set value to 12.0.8

**Review by SzymonKowalczyk — CHANGES_REQUESTED** [review]
You forgot to update db version - required for UBI currency automation

File: config/manager/manager.yaml (hunk 2 above)
Comment (marcin-ozog) [reply config/manager/manager.yaml]: thanks, updated

**Review by SzymonKowalczyk — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -15,7 +15,7 @@
 #
 
 # Current Operator version
-CSV_VERSION ?= 1.6.0
+CSV_VERSION ?= 1.7.0
 CSV_VERSION_DEVELOPMENT ?= development
 POSTGRESS_VERSION ?= 12.0.6
 OLD_CSV_VERSION ?= 1.5.0
```
Comment (kacperkrzyzak) [inline Makefile:21]: Update old csv version to 1.6.0 and postgres image version

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -605,5 +605,5 @@ spec:
   maturity: alpha
   provider:
     name: IBM
-  version: 1.6.0
+  version: 1.7.0
   replaces: ibm-licensing-operator.v1.5.0
```
Comment (kacperkrzyzak) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:609]: Update replaces to 1.6.0

**Review by kacperkrzyzak — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (marcin-ozog) [reply Makefile]: done

**Review by kacperkrzyzak — APPROVED** [review]

**Pull request merged**